app/src/main/java/cm/aptoide/pt/ApplicationModule.java

```diff
@@ -1002,12 +1013,14 @@ public ApplicationModule(AptoideApplication application, String aptoideMd5sum) {
       @Named("aptoideEvents") Collection<String> aptoideEvents,
       @Named("facebookEvents") Collection<String> facebookEvents,
       @Named("fabricEvents") Collection<String> fabricEvents,
-      @Named("flurryEvents") Collection<String> flurryEvents) {
+      @Named("flurryEvents") Collection<String> flurryEvents,
+      @Named("flurry") SessionLogger flurrySessionLogger) {
```

Comment:
I suggest using @Named("flurrySession") for a clearer understanding and not being confused with the EventLogger instance.